org/v1/org.proto

```diff
@@ -99,8 +99,20 @@ service Organization {
 
 // Request message for the Organization.CreateOrg rpc.
 message CreateOrgRequest {
+  // Required. Organization's email
   string email = 1;
+
+  // Required. Organization's description or company name
   string description = 2;
+
+  // Optional. User can decide to let the system generates the password.
+  string password = 3;
+
+  // Required. Start of the organization
```

Comment:
`month` and `option` shouldn't be exposed, I think. I need comments from @tuan78 as well.

Reply:
Noted chew. For the option, are we just going to use the standard template? For the month, cstool has an allowance of 1 year. I think this is for the start of the calculation. But, I think it would be okay to just use the current month.

Reply:
I remembered @tuan78 mentioned about making NTT the default version for new registrations, but I'm not sure on that. Let's wait for his comment(s).

Reply:
Sorry for late response, about the `month` we can use the current month and the `option` we use "standard" as default now